app/graphql/server.js

```diff
@@ -10,13 +10,19 @@ import { ApolloServerPluginLandingPageLocalDefault } from '@apollo/server/plugin
 const graphqlPath = dirname(fileURLToPath(import.meta.url))
 
 export const schema = makeExecutableSchema({
-  typeDefs: await loadFiles(join(graphqlPath, 'types')),
-  resolvers: await loadFiles(join(graphqlPath, 'resolvers')),
+  typeDefs: await loadFiles(join(graphqlPath, 'types'), {
+    recursive: true,
+    requireMethod: async path => import(pathToFileURL(path))
```

Comment:
Is this `requireMethod` needed? Shouldn't be necessary as loading async anyways

Reply:
Seems to be required to resolve imports on windows, can be removed when using linux